firedrake/utility_meshes.py

```diff
@@ -8,10 +8,10 @@
 
 from firedrake import VectorFunctionSpace, Function, Constant, \
     par_loop, dx, WRITE, READ
-from firedrake import mesh
+from firedrake import mesh, Mesh
```

Comment:
It would be more consistent with other uses if you just use `mesh.Mesh` instead of also importing `Mesh` directly.